src/Files.App/Views/Layouts/ColumnLayoutPage.xaml.cs

```diff
@@ -157,8 +156,8 @@ private void HighlightPathDirectory(ListViewBase sender, ContainerContentChangin
 			if (args.Item is ListedItem item && columnsOwner?.OwnerPath is string ownerPath
 				&& (ownerPath == item.ItemPath || ownerPath.StartsWith(item.ItemPath) && ownerPath[item.ItemPath.Length] is '/' or '\\'))
 			{
-				var presenter = args.ItemContainer.FindDescendant<Grid>()!;
-				presenter!.Background = this.Resources["ListViewItemBackgroundSelected"] as SolidColorBrush;
+				SetFolderBackground(args.ItemContainer as ListViewItem, this.Resources["ListViewItemBackgroundSelected"] as SolidColorBrush);
```

Comment:
Is there a way to make this theme aware to fix #12577?

Reply:
I think, the only option would be to listen to theme changes and based on that reapply the background. Unfortunately, there is no code behind way to create a ThemeResouce binding (at none that I am aware of).

Reply:
Is it worth adding logic for that or should we leave as is since restarting the app will use the correct theme?

Reply:
We've only had one bug report so it's not the highest priority, if it's a simple fix I guess we should move forward but otherwise it's not worth it if it requires a complex workaround.

Reply:
I think it's not too hard, we would probably only need an event listener for the actual theme changed event. I'll add a comment to the issue in case anyone else wants to pick it up (and make it easier for newcomers to contribute)

Reply:
Thanks!